includes/admin/settings.php

```diff
@@ -87,6 +91,45 @@ public function column_payment_method_value( $value, $affiliate ) {
 		return $value;
 	}
 
+	/**
+	 * Add a "Payment Method" column to the referrals screen.
```

Comment:
```suggestion
	 * Adds a "Payment Method" column to the referrals screen.
```
Use a third-person singular verb for DocBlock summaries.